cli/pcluster/cfnconfig.py

```diff
@@ -804,6 +804,26 @@ def __init_raid_parameters(self):  # noqa: C901 FIXME!!!
         except AttributeError:
             pass
 
+    def __check_fsx_updates(self, fsx_section, fsx_options):
+        """
+        Check FSx Parameters for updates.
+
+        :param fsx_section: name of FSx Section
+        :param fsx_options: dictionary of options
+        :return: True if updates were made else False
+        """
+        parameters = self.__get_stack_parameter("FSXOptions").split(",")
+        for i, key in enumerate(fsx_options):
```

Comment:
This might be simplified to the following (need to check if it really works :))
```
new_values = [self.__get_option_in_section(fsx_section, option) or "NONE" for option in fsx_options]
old_values = self.__get_stack_parameter("FSXOptions").split(",")
            
if new_values != old_values:
    self.__fail()
```

Reply:
Thanks! Confirmed it works